infra/README.md

```diff
@@ -86,24 +95,23 @@ gcloud run services update $CLOUD_RUN_SERVICE_NAME \
     --project $PROJECT_ID
 ```
 
-6. Verify that your set up.
+7. Verify your work.
```

Comment:
not sure this is the right words.. but Verify that you are set up or verify what you just did or something.